cmd/ct-malformed/main.go

```diff
@@ -0,0 +1,288 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+)
+
+func GETs(logURI string) {
+	tests := []struct {
+		Case           string
+		Endpoint       string
+		ExpectedStatus int
+	}{
+		// 	get-entries
```

Comment:
Can you add:

- extremely high start (and end somewhat greater than start)
- negative start and end
- end == start
- floating-point start / end

Similar ideas should apply to the other endpoints that have numeric inputs.

Reply:
> * end == start

This is a valid request for retrieving a single entry.

Reply:
so, status 200 then? :-D

Reply:
So I generally stayed away with doing 'positive' tests because here to do so you have to assume what the tree size is going to be, and even then only testing a single value for those only tells you it works for for inputs.

Reply:
(I was thinking yesterday that a nice tool to have might be something that starts from a empty log and does a set of operations to verify the log is operating as expected. Kind of like a test of expected test vectors.)